mousestyles/tests/test_kde.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+from scipy.stats.distributions import norm
+from mousestyles.kde import kde
+
+
```

Comment:
Please write a detailed comment explaining what this test is doing. Also write a simple comment above each `assert` statement

When writing the comment - always ask yourself - if I read this code in 6 hours, 6 days, 6 weeks etc would I be able to understand it just as clearly everytime? If not then add the explanatory comments to help guide you in the future :)